assets/dev/js/editor/container/container.js

```diff
@@ -521,6 +521,10 @@ export default class Container extends ArgsObject {
 		return elementor.userCan( 'design' ) && this.isEditable();
 	}
 
+	isGridContainer() {
+		return 'grid' === this.parent.args.model.attributes.settings.attributes.container_type;
```

Comment:
```suggestion
		return 'grid' === this.settings.get( 'container_type' );
```
@davseve Please check if this works?

Reply:
I think that we would like to check the parent state and not the container state itself.
Let's take the following examples:
**Parent**: grid. **Child**: widget - in this case we wat the grid to be resizable but we don't want its children to be resizable.
**Grandparent**: Grid. **Parent**: Flex. **Child**: Widget. - in this case, we would like the Grandparent to be resizable, the flex should not be resizable but the widget can be resizable. 

So AFAIU we need to check the parent value and if the parent is of type `grid`, than its direct child will not be resizable.

Reply:
Discussed on Slack to use `this.parent.settings.get()`